### DISCUSSION TITLE
Map store/Queue store configurations are not applied on node restart

### DISCUSSION THREAD

**Discussion by vishalpaalakurthi** [hazelcast/hazelcast#21396] [Q&A]
I am using hazelcast java client and creating map and queue configurations for instance.
map store or queue store configurations are configured properly on cluster startup and created external data store for them.
but, the created configurations are not working/executing when i restart the node.

Please help me if you are aware of it ?

Right now i am using kubernetes deployment model

**Comment by ldziedziul** [comment]
Persistence of the dynamic configuration is disabled by default, you have to enable it with `persistence-enabled` option (available in the Enterprise edition only), see https://docs.hazelcast.com/hazelcast/5.1/configuration/dynamic-config-persistence#enabling-persistence

**Reply by vishalpaalakurthi** [reply]
I am using load store for backups (alternative to native persistence provided by hazelcast) using mapstore and queuestore.

**Reply by TomaszGaweda** [reply]
@vishalpaalakurthi Can you please update the question with some simple reproducer: sample Java code with configuration and the description, how do you perform the restart. If this procedure you write causes the same same issue on reproducer code, we will be able to help you more precisely :)
